### DISCUSSION TITLE
Apple Watch steaming with Wyze bridge

### DISCUSSION THREAD

**Discussion by bellario89** [koush/scrypted#275] [Q&A] [answered]
As anyone had any luck with streaming to their Apple Watch using a Wyze bridge in between?

None of the transcoding options seem to work

Have both containers running on my synology

**Comment by cheme75** [comment, marked as answer]
Yes, wyze bridge on old win10 desktop to scypted in docker on old linux pc also running homebridge - 2 V2 cams and orig wyze db - stream to any of my homekit devices as long as bridge or scrypted pc is not locked up or crashed. I use apple tv as hub and ipad as backup hub. I rotate the db cam in the wyze bridge since it streams better and loads faster. I only use scrypted transcoding for one cam that is on its side but it slows down the stream a lot - had to use forced transcode debug in the homekit plugin to get the transpose to work. The watch stream is not rotated, only the DB is rotated from the bridge, but the rotated cam does stream in portrait view on iPhone and iPad and TV
